node_lambnik/scripts/destroy

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+function usage() {
+    echo -n "Usage: $(basename "${0}")
+Removes all published resources from AWS
+Options:
+    --help      Display this help text
+"
+}
+
+function main() {
+	echo "WARNING: you are about to destroy all your deployed resources!"
+	read -p "If this is what you want, type 'yes' to continue: " -r
+	echo
+	if [[ $REPLY =~ ^[yY][eE][sS]$ ]]
+	then
+
+		# Destroy deployed resources
+		# This command ALSO tried to destroy your lambda's IAM role.
+		# It will probably fail to do so, but it will delete the lambda
```

Comment:
This is a problem.
First, because it shows that we're still relying on hand-rolled AWS entities that aren't provisioned by the provisioning script, nor even documented, currently.
Second, because it means that if someone with more permissions runs it, they could delete the resources that we're relying on but don't have automatic provisioning for.
...which is what just happened.  Turns out I was running with my `research` AWS profile which has `AdministratorAccess`, so the role we've been using, `lambnik-tiler-dev`, is gone now.  We should probably restore that one by hand then figure out how to add role creation to the publish script before we merge this.  (Can Claudia create the necessary role?  It would seem weird for it to be deleting it by default if it's not able to create it.)

Reply:
Oh crap, I should have asked you about this issue right when I came across it, this is really my bad. What's happening now is that you can optionally configure a role in the `.env`, which I've been doing this whole time since one already existed to do the things lambnik needed to, and because I didn't have the permissions to create new roles. If you leave that off, Claudia will automatically create a new role for your lambda (assuming you have the permissions to do so).

I was able to find a gitter conversation on the Claudia repo in which some people discussed adding a tag to exclude the IAM role from being touched by `claudia destroy` for exactly this use case. Unfortunately it doesn't seem like anyone ever actually implemented it. What do you think the best course of action would be to make sure this script is safe to use?